### PR TITLE
Feat/hot fixes

### PR DESCRIPTION
**Pull request by joeandrews**
## Summary

This code is already deployed in SDK V1.11.0

## Description

Small updates to the event service to stop the SDK incorrectly selecting the block number of the last synced note for multiple metamask accounts. 

This PR also enables DAI permit in the SDK.

### CODE REVIEW

**Comment by joeandrews** [comment]
We are planning a wider refactor of this code today as its been the source of lots of balance bugs within the SDK.

File: packages/extension/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "**/*.scss"
     ],
     "license": "LGPL-3.0",
-    "version": "1.8.0",
+    "version": "1.11.0",
```
Comment (ArnSch) [inline packages/extension/package.json]: I believe this should actually be re-set to `0.0.0-semantically-released`.

File: packages/extension/src/background/services/EventService/index.js [hunk 2]
```diff
@@ -190,15 +190,18 @@ class EventService {
             const assets = await this.syncedAssets({ networkId });
```
Comment (ArnSch) [inline packages/extension/src/background/services/EventService/index.js:190]: A clean up of this file before merge would be good.

**Review by ArnSch — CHANGES_REQUESTED** [review]
Just a small cleanup request, otherwise 👍

**Comment by ArnSch** [comment]
:tada: This PR is included in version 0.18.1 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Comment by ArnSch** [comment]
:tada: This PR is included in version 1.7.1 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Comment by ArnSch** [comment]
:tada: This PR is included in version 1.12.0 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**